docs/getting-started.md

```diff
@@ -502,15 +496,15 @@ real UK population*"
    $ opensafely run run_all --force-run-dependencies
    ```
 
-   A new `input.csv` file will be created in the `output` folder. Open that
+   A new `input.csv.gz` file will be created in the `output` folder. Open that
    file (by left-clicking the filename in Visual Studio Code's Explorer, or
    software like Excel). This time, you'll see it now contains synthetic data: an age
    for 1000 randomly generated patients (we'll see shortly how this is defined).
```

Comment:
question: Does something need adding here about opening the `.csv.gz`? Also related to the removal of:

> This is what the `input.csv` should look like:

above.

Reply:
done